<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SCDF0</class>
 <widget class="QWidget" name="SCDF0">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>849</width>
    <height>583</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <property name="rightMargin">
    <number>9</number>
   </property>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <property name="leftMargin">
      <number>0</number>
     </property>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <property name="topMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QToolButton" name="button_zoom">
         <property name="text">
          <string>...</string>
         </property>
         <property name="icon">
          <iconset theme="go-home-symbolic">
           <normaloff>.</normaloff>.</iconset>
         </property>
         <property name="checkable">
          <bool>false</bool>
         </property>
         <property name="checked">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="toolButton">
         <property name="text">
          <string>...</string>
         </property>
         <property name="icon">
          <iconset theme="zoom-in-symbolic">
           <normaloff>.</normaloff>.</iconset>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QLabel" name="label_5">
         <property name="text">
          <string>x =</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="x_coord">
         <property name="text">
          <string>0</string>
         </property>
         <property name="textInteractionFlags">
          <set>Qt::LinksAccessibleByMouse|Qt::TextSelectableByKeyboard|Qt::TextSelectableByMouse</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>, y =</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="y_coord">
         <property name="text">
          <string>0</string>
         </property>
         <property name="textInteractionFlags">
          <set>Qt::LinksAccessibleByMouse|Qt::TextSelectableByKeyboard|Qt::TextSelectableByMouse</set>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QwtPlot" name="qwtPlot">
       <property name="canvasBackground">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>255</red>
          <green>255</green>
          <blue>255</blue>
         </color>
        </brush>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <property name="rightMargin">
      <number>0</number>
     </property>
     <item>
      <widget class="QGroupBox" name="groupBox">
       <property name="title">
        <string>Settings</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QCheckBox" name="calc_conj">
          <property name="text">
           <string>conj</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>FFT Length</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="psd_fftlen">
          <item>
           <property name="text">
            <string>64</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>128</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>256</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>512</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>1024</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>2048</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>4096</string>
           </property>
          </item>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string>Overlap</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="psd_overlap">
          <property name="maximum">
           <number>63</number>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_6">
          <property name="text">
           <string>Window Function</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="psd_window_function">
          <item>
           <property name="text">
            <string>Rechteck</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Hamming</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Blackman</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Blackman-Harris</string>
           </property>
          </item>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="button_log">
          <property name="text">
           <string>Log</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QwtPlot</class>
   <extends>QFrame</extends>
   <header>qwt_plot.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
